backend/app/routes.py

```diff
@@ -323,6 +323,72 @@ async def get_game_status():
         return jsonify({
             "answer": answer
         }), 200
+        
+    @app.route('/more_explanation', methods=['POST'])
+    async def more_explanation():
+        data = await request.get_json()
+        fen = data.get("fen")
+        question = data.get("question")
+        first_answer = data.get("first_answer")
+
```

Comment:
choose to use the fen or the previous request

Reply:
How can i get the fen from the previous request?